axolotl-web/package.json

```diff
@@ -20,7 +20,7 @@
     "core-js": "^2.6.11",
     "file-saver": "^2.0.2",
     "moment": "^2.27.0",
-    "ofcold-security-code": "^1.1.0",
+    "ofcold-security-code": "git+https://github.com/ofcold/security-code.git",
```

Comment:
I see the danger of non-reproducible builds by relying on the latest 'master'.
This library seems quite dead though, so should be a low-risk one.

Reply:
I am aware of that, that's the reason, why i opened https://github.com/ofcold/security-code/issues/8
Better would be to reimplement that part, because it's easily done.